rpc.go

```diff
@@ -126,21 +126,24 @@ func (r rpc) Channel(service string) transport.Channel {
 }
 
 func (r rpc) Start() error {
-	callServe := func(i transport.Inbound) func() error {
+	startInbound := func(i transport.Inbound) func() error {
 		return func() error {
 			return i.Start(r)
 		}
 	}
 
 	var wait sync.ErrorWaiter
 	for _, i := range r.inbounds {
-		wait.Submit(callServe(i))
+		wait.Submit(startInbound(i))
+	}
+
+	for _, o := range r.Outbounds {
+		wait.Submit(o.Start)
```

Comment:
What happens when the outbound errors during start? Does the rpc object blow up? Same thing on stop? Perhaps we should write a test that illustrates that.